Dockerfile

```diff
@@ -24,9 +24,7 @@ RUN go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.27.1 \
 WORKDIR /home/deepfence/src/YaRadare
 COPY . .
 RUN make clean \
-    && make all \
-    && cd /home/deepfence \
-    && git clone https://github.com/deepfence/yara-rules
+    && make all 
```

Comment:
I think image should have rules in-built.
This is required when console is running in air-gapped environment